### ISSUE TITLE
Fail to run all examples

### ISSUE THREAD

**Issue by mashnode** [charmbracelet/bubbletea#16]
For codes in https://github.com/charmbracelet/bubbletea/tree/master/examples
```
$ grun
examples/http
# examples/http
./main.go:28:21: not enough arguments in call to tea.NewProgram
	have (model)
	want (tea.Init, tea.Update, tea.View)
```

Can you please fix this for beginner like me?

**Comment by mashnode** [comment]
I found that I removed this line from go.mod

```
replace github.com/charmbracelet/bubbletea => ../
```

It works after I added it back. So why this error happens? It's kinda strange.

**Comment by meowgorithm** [comment]
Hi! Sorry about that. This is be fixed in `master` (by b8bc66ed6f57f72bc237f7d55464266f3c542ac0).

We changed the Bubble Tea API slightly in v0.12.0, and the version in the examples was v0.11.1. The reason things worked with the `replace` directive is because it overrides the version specified in `go.mod` and instead tells Go to use the local version in the parent directory.
